src/App.js

```diff
@@ -1,14 +1,34 @@
 import './App.css';
 import { useFeaturedBanners } from './utils/hooks/useFeaturedBanners';
+import Header from "./components/Header/Header";
 import HomePage from './pages/HomePage';
+import ProductListPage from "./pages/ProductListPage/ProductListPage";
+import {useState} from "react";
+
+const PAGES = {
+  home: 'home',
+  productList: 'productList'
+}
 
 function App() {
+  const [currentPage, setCurrentPage] = useState(PAGES.home);
   const { data, isLoading } = useFeaturedBanners();
   console.log(data, isLoading);
 
+  const onClickSetPage = (page = 'home') => {
+      setCurrentPage(page)
+  }
+
   return (
     <div className="App">
-      <HomePage />
+      <Header onClickToggleMenu={onClickSetPage} />
+      {
+        currentPage === 'home' ? (
+          <HomePage onClickToggleMenu={onClickSetPage} />
+        ) : (
+          <ProductListPage />
+        )
+      }
```

Comment:
```suggestion
      }
      <Footer/>
```
We can move the `Footer` component here, since it belongs to the general structure of the app, and this way we don't repeat code in `HomePage` and `ProductListPage`